sefaria/export.py

```diff
@@ -726,7 +726,11 @@ def _import_versions_from_csv(rows, columns, user_id):
     from sefaria.tracker import modify_bulk_text
 
     index_title = rows[0][columns[0]]  # assume the same index title for all
-    index_node = Ref(index_title).index_node
+    index = Index().load({'title': index_title})
```

Comment:
Just to make sure I understand - when passing a title to `Ref(..)` it will try to come up with any match, including non-unique ones, based on the title, making it inaccurate?

Reply:
Right, for example Ref('Gen') will work.